web/blueprints/finance/__init__.py

```diff
@@ -830,6 +830,7 @@ def transactions_unconfirmed_json():
 
         items.append(
             {
+                'id': transaction.id,
```

Comment:
Not quite sure how this went through `darker`; the implicit convention for new code is double quotes instead of single quotes.
```suggestion
"id": transaction.id,
```

Reply:
I haven't noticed that github trimmed the whitespace in the copy'n'paste process. Fixing this in a commit.
I now realized why `darker` did not catch this: the `pre-commit` hook only ran on the last commit of this PR instead of the whole range. I've fixed this in 999e1f02b79c4f26b782229e982b0da0c3baa127.